build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1412,6 +1429,11 @@ message GetActionResultRequest {
   // `output_files` (DEPRECATED since v2.1) in the
   // [Command][build.bazel.remote.execution.v2.Command] message.
   repeated string inline_output_files = 5;
+
+  // A list of acceptable encodings to used for inlined data.  Must be
+  // `IDENTITY` (or unspecified), or one or more of the compressors
+  // supported by the server.
```

Comment:
I think that this phrasing is far too strict. Why does it need to contain compressors supported by the server? Couldn't the server just ignore the ones it doesn't support?

Furthermore, what's the point in making IDENTITY explicit in this list? I would do it the other way around: forbid IDENTITY from being listed explicitly, allowing servers to always pick this compressor for responses.

Reply:
s/Must/Should/ sounds wise. Note this is just a quick first pass, for comparison with #203.

If a client specifies `IDENTITY` here, the server knows it's not an old client, and can inline uncompressed data in the Blob instead of in the legacy field.

Reply:
But this PR doesn't contain Blob.

Reply:
You're right, this is probably something that can be dropped from this option.